owid/datautils/dataframes.py

```diff
@@ -322,3 +322,74 @@ def multi_merge(
         merged = pd.merge(merged, df, how=how, on=on)
 
     return merged
+
+
+def map_series(
+    series: pd.Series,
+    mapping: Dict[Any, Any],
+    make_unmapped_values_nan: bool = False,
+    warn_on_missing_mappings: bool = False,
+    warn_on_unused_mappings: bool = False,
+    show_full_warning: bool = False,
+) -> pd.Series:
+    """Map values of a series given a certain mapping.
+
+    This function does almost the same as
+    > series.map(mapping)
+    However, map() translates values into nan if those values are not in the mapping, whereas this function allows to
```

Comment:
FYI there's [pd.Series.update](https://pandas.pydata.org/docs/reference/api/pandas.Series.update.html) method that does this. Just saying, your function is much more powerful, so don't change anything!

Reply:
Oh, I didn't know that method! Maybe it can be used to speed up the replacement of nans. Feel free to change it if you think that could be the case.

Reply:
I don't think it's gonna be faster. It's also in-place only, so not worth it. Go ahead and merge it!